packages/@vue/cli-plugin-pwa/lib/HtmlPwaPlugin.js

```diff
@@ -111,13 +111,18 @@ module.exports = class HtmlPwaPlugin {
               rel: 'manifest',
               href: getTagHref(publicPath, manifestPath, assetsVersionStr)
             }
-          ),
-          makeTag('meta', {
-            name: 'theme-color',
-            content: themeColor
-          })
+          )
         )
 
+        if (themeColor !== null) {
```

Comment:
```suggestion
        if (themeColor != null) {
```

Need to check against `undefined`, so use `!=` instead of `!==`